src/docs/merge-patch.md

```diff
@@ -54,10 +54,11 @@ Note that this option only works if the patch applies cleanly on the release bra
 1. Open the CL you want to back-merge.
 1. Select "Cherry pick" from the extended menu (three vertical dots in the upper right corner).
 1. Enter "refs/branch-heads/*X.X*" as destination branch (replace *X.X* by the proper branch).
+1. In case of merge conflict, please also go ahead and create the CL
```

Comment:
I think this step should be moved down to *after* modifying the commit message (which still happens before actually creating the cherry-pick, thus before the merge conflicts are detected).

As there is already an item below about resolving conflicts, maybe we can also just drop this item here completely.

Reply:
Good catch - just tried the flow and you're right. Adjusted the ordering to reflect the flow